src/components/policy/policy_external/test/update_status_manager_test.cc

```diff
@@ -72,18 +73,51 @@ class UpdateStatusManagerTest : public ::testing::Test {
   void TearDown() OVERRIDE {}
 };
 
-// TODO(AKutsan) : APPLINK-31222 use miliseconds as parameter of OnUpdateSentOut
-// function
-TEST_F(
-    UpdateStatusManagerTest,
-    DISABLED_OnUpdateSentOut_WaitForTimeoutExpired_ExpectStatusUpdateNeeded) {
+namespace {
+class WaitAsync {
+ public:
+  WaitAsync(uint32_t count, uint32_t timeout)
+      : count_(count), timeout_(timeout), cond_var_() {}
+
+  void Notify() {
+    count_--;
+    cond_var_.NotifyOne();
+  }
+
+  void Wait(sync_primitives::AutoLock& auto_lock) {
+    while (count_ > 0) {
+      sync_primitives::ConditionalVariable::WaitStatus wait_status =
+          cond_var_.WaitFor(auto_lock, timeout_);
+      if (wait_status == sync_primitives::ConditionalVariable::kTimeout) {
+        break;
+      }
+    }
+  }
+
+ private:
+  int count_;
+  uint32_t timeout_;
+  sync_primitives::ConditionalVariable cond_var_;
+};
+}
+
+ACTION_P(NotifyAsync, waiter) {
+  waiter->Notify();
+}
+
+TEST_F(UpdateStatusManagerTest,
+       OnUpdateSentOut_WaitForTimeoutExpired_ExpectStatusUpdateNeeded) {
   // Arrange
+  sync_primitives::Lock lock;
+  sync_primitives::AutoLock auto_lock(lock);
+  WaitAsync waiter_(3, 2 * k_timeout_);
```

Comment:
@LitvinenkoIra , please avoid 'magic numbers'. Shoul better do in such way: 
`const uint32_t count = 3;
const  uint32_t timeout = 2 * k_timeout_;
WaitAsync waiter_(count, timeout);`
Also `waiter` is not private member